GPT/gpt.py

```diff
@@ -200,12 +199,12 @@ def gpt_insert_response(
             case "above":
                 actions.key("left")
                 actions.edit.line_insert_up()
-                actions.user.paste(result)
+                actions.user.insert_snippet(result)
```

Comment:
insert_snippet falls back to paste, so it should be equivalent for most use cases

Reply:
never mind-it turns out that it's much slower so I don't want to use this unless necessary